src/socket.c

```diff
@@ -335,6 +335,16 @@ static int connSocketAddr(connection *conn, char *ip, size_t ip_len, int *port,
     return C_ERR;
 }
 
+static int connSocketIsLocal(connection *conn)
+{
+    char cip[NET_IP_STR_LEN + 1] = { 0 };
+
+    if (connSocketAddr(conn, cip, sizeof(cip) - 1, NULL, 0) == C_ERR)
```

Comment:
the old code used to call connAddrPeerName, setting `remote` set to `1`.
here you call it with `remote` set to 0 (effectively castrating this feature)

Reply:
OK